app/assets/stylesheets/components/find/_call-out-box.scss

```
@@ -0,0 +1,16 @@
+.app-find-pink-callout {
+  border-left-color: #d43681;
+  background-color: #f3f2f1;
+  padding: 15px;
+  border-left-width: 5px;
+  min-height: 130px;
```

Can we split out these classes for reusability please?

```suggestion
.govuk-inset-text.app-callout {
  background-color: #f3f2f1;
  padding: 15px;
  border-left-width: 5px;
  min-height: 130px;
  
  &.app-callout--pink {
    border-left-color: #d43681;
  }
}
```

This way we can extend the callout component to add different colours, along with a base colour - In this particular case, it would be the default inset text colour.

The `.app-callout` css is also dependent on the `.govuk-inset-text` class, we shouldn't be able to use `app-callout` without the `govuk-inset-text` css class present, otherwise it would look broken, I _think_, unless it is standalone?